src/ozone/ozone.py

```diff
@@ -213,6 +245,30 @@ def get_city_air(
             df = pandas.concat([df, pandas.DataFrame(row)], ignore_index=True)
         return self._format_output(data_format, df)
 
+    def get_multiple_location_air(
+        self,
+        locations: List[Tuple],
+        data_format: str = "df",
+        df: pandas.DataFrame = pandas.DataFrame(),
+        params: List[str] = [""],
+    ) -> pandas.DataFrame:
+        """Get multiple locations air quality data
+
+        Args:
+            locations (list): A list of pair (latitude,longitude) to get data for.
+            data_format (str): File format. Defaults to 'df'. Choose from 'csv', 'json', 'xslx'.
+
+        Returns:
+            pandas.DataFrame: The dataframe containing the data. (If you
+            selected another data format, this dataframe will be empty)
+        """
+        for loc in locations:
+            # This just makes sure that it's always a returns a pd.DataFrame. Makes mypy happy.
+            df = pandas.DataFrame(self.get_lat_lon_air(loc[0], loc[1], df=df, params=params))
```

Comment:
I think you forgot to rename the `get_lat_long_air` method here. Rename to `get_coordinate_air`